Homework-1/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="styles.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css">
+</head>
+
+<body onload="loadReviews()">
+  <main>
+    <h2>TV shows APP</h2>
+    <div class="show-info">
+      <img src="./brooklyn99.jpg" />
```

Comment:
It is usually smart to separate your code and images. Usually, you would place your images in a separate folder called 'assets' or something similar. Please do this.